revert/swrs/transform/function/load_fuel_mapping.sql

```diff
@@ -155,12 +155,14 @@ $function$
         INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Flared Natural Gas CO2', 85);
         INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Flared Natural Gas CH4', 86);
         INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Flared Natural Gas N2O', 87);
-        INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Vented Natural Gas', 88);
+        INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Vented Natural Gas CH4', 88);
```

Comment:
should this revert file be updated?

Reply:
I think this revert file is fine? It's undoing the name change, which is what it should do?